src/python/ddapp/startup.py

```diff
@@ -166,6 +166,14 @@
 poseCollection = PythonQt.dd.ddSignalMap()
 costCollection = PythonQt.dd.ddSignalMap()
 
+if 'fixedBaseArm' in drcargs.getDirectorConfig()['userConfig']:
+    ikPlanner.fixedBaseArm = True
+if 'disableComponents' in drcargs.getDirectorConfig():
+    for component in drcargs.getDirectorConfig()['disableComponents']:
+        print "Disabling", component
+        exec(component + " = False")
```

Comment:
slightly better:

locals()[component] = False

But in the follow up commit we could introduct an Options struct instead of having the useXX global variables.
For example:

from ddapp.fieldcontainer import FieldContainer

    opts = FieldContainer(
      useFootsteps = True,
      usePlanning = True
    )

    # now we can access opts.usePlanning